src/lib/y2storage/guided_proposal.rb

```diff
@@ -205,17 +205,58 @@ def candidate_devices_with_empty_partition_table(devicegraph)
       devices.select { |d| d.partition_table && d.partitions.empty? }
     end
 
+    # Returns the target sizes to make the proposal
+    #
+    # @return [Array<Symbol>]
+    def target_sizes
+      [:desired, :min]
+    end
+
     # Copy of the original settings including some calculated and necessary
     # values (mainly candidate_devices), in case they were not present
     #
     # @return [ProposalSettings]
     def populated_settings
```

Comment:
NP: I would change the description of the method, because now, it does not add extra info to the settings by itself. I would say that "populated settings" are the settings used by each attempt of proposal, and some values could be "populated" during the process, for example, the candidate devices or the root device.